pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -27,6 +27,18 @@ def test_roundtrip_pickle_with_tz(self):
         unpickled = tm.round_trip_pickle(index)
         tm.assert_index_equal(index, unpickled)
 
+    @pytest.mark.parametrize("test_input", [
+        DatetimeIndex(['2013-01-24 15:01:00']),
```

Comment:
Instead, you could leave the `DatetimeIndex` construction in the test, and parametrize over the dtypes and pass that into the constructor.